dotnet.md

```diff
@@ -0,0 +1,38 @@
+# .NET on Graviton
+
+.NET is a free, cross-platform, open source developer platform for building many different types of applications. .NET applications are written in the C#, F#, or Visual Basic programming language. Code is compiled into a language-agnostic Common Intermediate Language (CIL). Compiled code is stored in assemblies—files with a .dll or .exe file extension. For more information, please see [what is .NET](https://dotnet.microsoft.com/learn/dotnet/what-is-dotnet).
```

Comment:
We can't just copy the text from a different website

Reply:
Rewrote copy.